### ISSUE TITLE
JS ERROR Error: More than one view found in list template! Nesting level: 0

### ISSUE THREAD

**Issue by kamok** [telerik/nativescript-ui-feedback#240]
### Did you verify this is a real problem by searching [Stack Overflow](http://stackoverflow.com/questions/tagged/nativescript)?
Yes. 

### Tell us about the problem
_Please, ensure your title is less than 63 characters long and starts with a capital
letter._
On your demo application, in https://github.com/telerik/nativescript-ui-samples-angular, the sample app throws an error and crashes. I've had 1 other developer on my team confirm this, as well as another dev on your Slack confirm this. 
### Which platform(s) does your issue occur on?
Both, latest versions
### Please provide the following version numbers that your issue occurs with:
- **Progress NativeScript UI version:** (the version of nativescript-telerik-ui/-pro in the `package.json`)     `"nativescript-telerik-ui-pro": "3.0.1"`
- **CLI:** (run `tns --version` to fetch it) `3.1.2`
- **Cross-platform modules:** (the version of tns-core-modules in the `package.json`) `    "tns-core-modules": "3.1.0"`
- **Runtime(s):** (the `"tns-android"` and `"tns-ios"` properties in the `package.json`)
```
    "tns-ios": {
      "version": "3.1.0"
    },
    "tns-android": {
      "version": "3.1.0"
    }
```
### Please tell us how to recreate the issue in as much detail as possible.
1. Start the application ...
2. Crash ...
### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.
_(You can paste entire code snippets or attach a runnable project)_

**Comment by kamok** [comment]
I tried to debug with both VSCode + the plugin and doing `tns debug ios`. I'm unable to use those tools to do it successfully. I'm sure its my inexperience with your tech.

**Comment by VladimirAmiorkov** [comment]
This issue has bee reported and resolved in our latest version **3.0.2**. For more details please refer this thread: https://github.com/telerik/nativescript-ui-feedback/issues/238
